test/helpers/contents_list_helper_test.rb

```diff
@@ -0,0 +1,21 @@
+require 'test_helper'
+
+class ContentsListHelperTest < ActionView::TestCase
+  test "it wraps the number and text in span elements" do
+    input = '<a data-track-category="contentsClicked" data-track-action="leftColumnH2" data-track-label="run-an-effective-welfare-system-that-enables-people-to-achieve-financial-independence-by-providing-assistance-and-guidance-into-employment" href="#run-an-effective-welfare-system-that-enables-people-to-achieve-financial-independence-by-providing-assistance-and-guidance-into-employment">1. Run an effective welfare system that enables people to achieve financial independence by providing assistance and guidance into employment</a>'
```

Comment:
They're not - I'd simply copied and pasted a real world example. Will trim it down to be more readable.